src/js/tabs/pid_tuning.js

```diff
@@ -2341,8 +2341,9 @@ TABS.pid_tuning.changeRatesType = function(rateTypeID) {
 
     self.currentRatesType = rateTypeID;
 
-    if (self.currentRatesType !== RC_tuning.rates_type) {
+    if ((self.currentRatesType !== RC_tuning.rates_type) && (!dialogRatesType.hasAttribute('open'))) {
```

Comment:
The `(!dialogRatesType.hasAttribute('open'))` seems to be going too far now - it also covers `sameRatesType = false` and the `else` case.